app/core/config.py

```diff
@@ -36,7 +36,7 @@
     cast=CommaSeparatedStrings,
     default="*",
 )
-with open("/data/blast_config.json") as f:
+with open("../data/blast_config.json") as f:
```

Comment:
is this correct?

Reply:
It'll look from inside the app directory into the data directory that's in the repo; should be correct 

<img width="220" alt="image" src="https://github.com/user-attachments/assets/72b23341-aaf0-4812-9f97-c8d42be89e57">

Reply:
did that for local setup/testing shouldn't be this way. Corrected